object-manipulation-2/src/omit.js

```diff
@@ -0,0 +1,28 @@
+/* exported omit */
+function omit(source, keys) {
+  var newObj = {};
+  var tempCount = 0;
+  for (var i in source) {
+    for (var j in keys) {
+      if (i === keys[j]) {
+        tempCount++;
+      }
+    }
+    if (tempCount === 0) {
+      newObj[i] = source[i];
+    }
+    tempCount = 0;
+  }
```

Comment:
You can apply the same revision logic I provided for `defaults()` here, just in reverse